src/modules/openCageWrapper.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const getCity = async (lat, long) => {
+  var apiKey = "09a3f3539e5949878aeb212e4bcce44a";
```

Comment:
Move the api key to .env

Reply:
Somehow thats not working, Oliver tried it as well. But its a open sourced API that Ive recived on email, Im okay that its living free

Reply:
Okay, but I would like to see HOW the solution you claim did not work, look like. Please paste some code in a comment.

Reply:
The problem was that you couldn't have the api key with quotes in the .env file like the other keys had. Now it works

Reply:
:)